### PR TITLE
[16.0][MIG] fleet_vehicle_calendar_year: Migration to 16.0

### PR DESCRIPTION
**Pull request by kaynnan**

### CODE REVIEW

**Review by marcelsavegnago — APPROVED** [review]
LGTM

File: fleet_vehicle_calendar_year/models/fleet_vehicle.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+from odoo import fields, models
+
+
+class FleetVehicle(models.Model):
+
+    _inherit = "fleet.vehicle"
+
+    calendar_year = fields.Char(
+        track_visibility="onchange",
```
Comment (ivantodorovich) [inline fleet_vehicle_calendar_year/models/fleet_vehicle.py]:
```
2023-04-06 04:27:02,178 238 WARNING odoo odoo.fields: Field fleet.vehicle.calendar_year: unknown parameter 'track_visibility', if this is an actual parameter you may want to override the method _valid_field_parameter on the relevant model in order to allow it 
```

```suggestion
        tracking=True,
```

Comment (kaynnan) [reply fleet_vehicle_calendar_year/models/fleet_vehicle.py]: Completed changes, thanks

**Review by ivantodorovich — APPROVED** [review]

**Pull request merged**